### ISSUE TITLE
amp-imgur gifs behave abusing on mobile

### ISSUE THREAD

**Issue by jimbojetlag** [ampproject/amphtml#20437]
This is not a bug report per se, but more of an ethical problem with Imgur abusing amp as clickbait on mobile.

You can test this with an amp-imgur tag with a gif source on a mobile device. The tag will render a native video player, however when the the user taps on the click button the video does not play. Instead it opens the original url on imgur.com in a new browser tab, which is not the expected behavior.

Since AMP is a sandbox, we can only use the official amp-imgur tag to display content from imgur. Hence imgur is using the AMP channel to systematically render clickbait content on mobile.

Does AMP have a guideline or a policy about this?

**Comment by aghassemi** [comment]
@jimbojetlag Thanks for the report. That seems to be the way amp-imgur embeds work on the web ( https://help.imgur.com/hc/en-us/articles/211273743-Embed-Unit ). 

However for me all videos auto play and `pause`, `play` icons do not take me to imgur ( clicking anywhere else does however), so doesn't seem too bad. Do you have a Url I can look at where video does not autoplay and `play` button takes the user to imgur?

**Comment by aghassemi** [comment]
yeah i am not sure if we can do much about that. we can prevent opening a new window for the component but that breaks useful functionality in other cases.

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @aghassemi Please add an appropriate category.

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @aghassemi Please add an appropriate category.

**Comment by aghassemi** [comment]
Closing this as it is outside of our control how inside of imgur iframe works.
